lms/djangoapps/grades/new/course_grade_factory.py

```diff
@@ -187,6 +189,12 @@ def _update(user, course_data, read_only):
             course_key=course_data.course_key,
             deadline=course_data.course.end,
         )
+        if course_grade.passed is True:
+            LEARNER_COURSE_PASSING_GRADE.send_robust(
```

Comment:
If this is better as a downstream signal, fair. Was just trying to keep the calculations to a minimum.